### ISSUE TITLE
Do not remove Elementor Popup Styles when enabling Remove Unused CSS

### ISSUE THREAD

**Issue by dotrahat** [wp-media/wp-rocket#4209]
**Before submitting an issue please check that you’ve completed the following steps:**
✅ - Made sure you’re on the latest version
✅ - Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
Popups are not initially visible when page is opened. These only appear when triggered through a condition such as a timer, click on a button or via onclick function. Elementor Pro is one of the most popular plugin having more than 5+ Million Active Installs. And it has functionality of popup builder which is used almost by every Elementor Pro user.

Because the Popup doesn't appear with the page load, so its CSS is also removed from the page when 'Remove Unused CSS' feature of WP Rocket is Activated. In this way, the popup doesn't work / appear the way it should - and it should not happen.

**To Reproduce**
1. Install Elementor Pro
2. Create a Popup with Elementor
3. Set whatever condition to show it - entire site & trigger in a few seconds say or via onclick
4. Install & Activate WP Rocket , Enable 'Remove Unused CSS' Feature
5. After 'used css' is generated, Open any website page in incognito and see the popup - it appends at the end of page

**Expected behavior**
Popup should appear as it does when Remove Unused CSS is DISABLED.

**Screenshots**
![Screenshot_1732](https://user-images.githubusercontent.com/61520096/127519477-be0087b0-24b8-4b5d-9844-f6dafaf3468b.png)
![Screenshot_1731](https://user-images.githubusercontent.com/61520096/127519501-3b36cc11-1748-496b-b2a2-013aa8e01f00.png)
![Screenshot_1730](https://user-images.githubusercontent.com/61520096/127519530-e2f16583-e04c-4cb6-a028-b1ccc81cda33.png)

**Additional context**
I tried to exclude all classes of Elementor Popup but it doesn't work fine still. Only worked fine when I excluded entire CSS file of that popup that Elementor generates. Classes are:

.dialog-widget
.dialog-lightbox-widget
.dialog-type-buttons
.dialog-type-lightbox
.elementor-popup-modal

These classes are in every popup. In addition to these, each popup has a CSS ID as well that goes like #elementor-popup-modal-{ID}

CSS Files for Popups go in directory like : https://domain.com/wp-content/uploads/elementor/css/post-{ID}.css

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by mehedihasanziku** [comment]
Yes, it's an reproduceable issues with all popup/lightbox.

WProcket not have an solution for it unless doing it manually. so better approach will be if they support wildcard to rucss safe list or add such wildcard to their own safelist.

**Comment by NataliaDrause** [comment]
Related ticket: https://secure.helpscout.net/conversation/1583111685/283071/

The video light box popup doesn't work when _Remove Unused CSS_ is enabled.
The following selectors added to the Safelist resolved the issue:
```
.dialog-widget-content
.dialog-lightbox-widget-content
.dialog-header
.dialog-lightbox-header
.dialog-message
.dialog-lightbox-message
.dialog-buttons-wrapper
.dialog-lightbox-buttons-wrapper
.dialog-widget
.dialog-lightbox-widget
.dialog-type-buttons
.dialog-type-lightbox
.dialog-close-button
.dialog-lightbox-close-button
.elementor-video-container
.elementor-lightbox-prevent-close
.elementor-fit-aspect-ratio
.elementor-lightbox
.elementor-aspect-ratio-169
```
More details in the note: https://secure.helpscout.net/conversation/1583111685/283071/#thread-4637141095

**Comment by mehedihasanziku** [comment]
@NataliaDrause yes, but it will not fix all styling issues, because after js loaded or popup triggered, it's wrap within this id `"#elementor-popup-modal-*"` and if not exist it breaks. now here `*`(id) is unique for every popup.
